platform/viewer/public/config/local_static.js

```diff
@@ -12,9 +12,9 @@ window.config = {
       sourceName: 'dicomweb',
       configuration: {
         name: 'DCM4CHEE',
-        wadoUriRoot: '/dicomweb',
-        qidoRoot: '/dicomweb',
-        wadoRoot: '/dicomweb',
+        wadoUriRoot: 'http://localhost:5000/dicomweb',
```

Comment:
And the localhost:5000 prevents reloads/serving up a fairly large dicomweb directory within the webpack config.

Reply:
Can you add this comment to the source code? this is a `js` file and we can add comments there